### PR TITLE
added plugin mqtt_for_psucontrol

### PR DESCRIPTION
**Pull request by oerkel47**

### CODE REVIEW

File: mqtt_for_psucontrol.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+layout: plugin
+id: mqtt_for_psucontrol
+title: MQTT for PSU Control
+description: Use MQTT and/or HomeAssistant to control the switch you use with the PSU Control plugin
+archive: https://github.com/oerkel47/OctoPrint-MQTT-for-PSUcontrol/archive/main.zip
+homepage: https://github.com/oerkel47/OctoPrint-MQTT-for-PSUcontrol
+source: https://github.com/oerkel47/OctoPrint-MQTT-for-PSUcontrol
+author: oerkel47
+license: AGPLv3
+featuredimage: https://github.com/oerkel47/OctoPrint-MQTT-for-PSUcontrol/blob/main/screenshot_settings.PNG
```
Comment (cp2004) [inline mqtt_for_psucontrol.md:11]: Screenshots/Featured images should be stored in the plugin repository, under /assets/img/plugins/... etc, not served from a 3rd party

**Review by cp2004 — CHANGES_REQUESTED** [review]

File: mqtt_for_psucontrol.md (hunk 1 above)
Comment (oerkel47) [reply mqtt_for_psucontrol.md]: Sorry, thought github was okay. Should be fixed!

**Comment by jneilliii** [comment]
Because of it's tight integration with PSU Control, I think @kantlivelong would be the best to review/approve this one.

**Review by kantlivelong — APPROVED** [review]

**Comment by oerkel47** [comment]
Yay, my first plugin! Thanks everyone.

**Comment by kantlivelong** [comment]
@oerkel47 looks like we missed one issue but I just fixed it. The md file was in the root directory instead of /_plugins/

Fixed via 39e5c8cba89331e830f86802dcea25fdf6f3cbed

**Comment by oerkel47** [comment]
Thanks..!

Just checked out the plugin page and noticed the featured image is not showing up anywhere. Is the feature broken or did I misunderstand it? Anyway..I would like to add this "missing" image to the picture settings. Can I just create another pull request for an update? This time in the correct folder..I swear ;D

**Comment by cp2004** [comment]
The featured image is used for things like the feed in OctoPrint itself and social media stuff, eg. https://mobile.twitter.com/OctoPrint3D/status/1392562165366788098

You can create as many PRs as you like, to update the plugin page when necessary.

**Comment by oerkel47** [comment]
Cool, thanks for the twitter link! That clears it up.

**Pull request merged**